modules/apps/transfer/keeper/relay_forwarding_test.go

```diff
@@ -19,39 +21,54 @@ import (
 	ibctesting "github.com/cosmos/ibc-go/v8/testing"
 )
 
-func (suite *KeeperTestSuite) setupForwardingPaths() (pathAtoB, pathBtoC *ibctesting.Path) {
-	pathAtoB = ibctesting.NewTransferPath(suite.chainA, suite.chainB)
-	pathBtoC = ibctesting.NewTransferPath(suite.chainB, suite.chainC)
-	pathAtoB.Setup()
-	pathBtoC.Setup()
-	return pathAtoB, pathBtoC
-}
-
-type amountType int
-
 const (
 	escrow amountType = iota
 	balance
 )
 
-func (suite *KeeperTestSuite) assertAmountOnChain(chain *ibctesting.TestChain, balanceType amountType, amount sdkmath.Int, denom string) {
-	var total sdk.Coin
-	switch balanceType {
-	case escrow:
-		total = chain.GetSimApp().TransferKeeper.GetTotalEscrowForDenom(chain.GetContext(), denom)
-	case balance:
-		total = chain.GetSimApp().BankKeeper.GetBalance(chain.GetContext(), chain.SenderAccounts[0].SenderAccount.GetAddress(), denom)
-	default:
-		suite.Fail("invalid amountType %s", balanceType)
+type (
+	ForwardingTestSuite struct {
+		testifysuite.Suite
```

Comment:
I find this syntax for struct declaration a lot less readable than the standard one definition per line. We are also only declaring a single struct here so I think there's no reason not to use the standard

```golang
type ForwardingTestSuite struct{ ... }
```

syntax